crates/rpc/ipc/src/server/mod.rs

```diff
@@ -244,16 +269,87 @@ pub(crate) struct ServiceData {
     pub(crate) method_sink: MethodSink,
 }
 
+/// Similar to [`tower::ServiceBuilder`] but doesn't
+/// support any tower middleware implementations.
+#[derive(Debug, Clone)]
+pub struct RpcServiceBuilder<L>(tower::ServiceBuilder<L>);
```

Comment:
do we need this type?
looks like this is the same as the jsonrpsee type, so we can use that instead?

Reply:
Yes, they are the same, but `RpcServiceBuilder::service` is `pub(crate)`, and I have no other way to setting inner service other than this method. 
https://github.com/paritytech/jsonrpsee/blob/98675a06ee0d386256954c4fa21b3fb02345d045/server/src/middleware/rpc/mod.rs#L102